nixos/modules/services/audio/navidrome.nix

```
@@ -96,6 +96,10 @@ in
             mode = "700";
             inherit (cfg) user group;
           };
+          "${cfg.settings.MusicFolder or (WorkingDirectory + "/music")}"."d" = {
+            mode = "700";
+            inherit (cfg) user group;
```

I don't think we should modify the mode or ownership if the music directory already exists.
```suggestion
            mode = ":700";
            user = ":${cfg.user}";
            group = ":${cfg.group}";
```

---

Oh that's great, [systemd tempfiles.d](https://www.freedesktop.org/software/systemd/man/latest/tmpfiles.d.html) is new to me!
> Optionally, if prefixed with ":", the configured access mode is only applied when creating new inodes, and if the inode the line refers to already exists, its access mode is left in place unmodified.